Dockerfile.dapper

```diff
@@ -0,0 +1,29 @@
+FROM golang:1.13.4-alpine3.10
```

Comment:
Should this be in line with k3s? `FROM golang:1.15.5-alpine3.12`

Reply:
fixed, Thank you